matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/pushers/PushersService.kt

```diff
@@ -128,4 +107,62 @@ interface PushersService {
      * Get the current pushers
      */
     fun getPushers(): List<Pusher>
+
+    data class HttpPusher(
+
+            /**
+             * This is a unique identifier for this pusher. The value you should use for
+             * this is the routing or destination address information for the notification,
+             * for example, the APNS token for APNS or the Registration ID for GCM. If your
+             * notification client has no such concept, use any unique identifier. Max length, 512 chars.
+             * If the kind is "email", this is the email address to send notifications to.
```

Comment:
I think with this Service API change, the kind should not be "email", so maybe remove this line?